### PR TITLE
Adding application metrics

### PR DESCRIPTION
**Pull request by Alessar**
Fixes issue #876
Adding some metrics to eventnative application

### CODE REVIEW

File: server/metrics/destinations.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	successDestinations *prometheus.Counter
+	errorDestination    *prometheus.Counter
+)
+
+func initDestinations() {
+	successDestinations = NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "eventnative",
+			Subsystem: "destinations",
+			Name:      "success",
```
Comment (absorbb) [inline server/metrics/destinations.go:17]: please clarify metrics names and corresponding methods.
init_success, init_error

That applies for sources to

File: server/authorization/service.go [hunk 2]
```diff
@@ -182,12 +184,15 @@ func (s *Service) GetTokenID(tokenFilter string) string {
 func (s *Service) updateTokens(payload []byte) {
 	tokens, err := parseFromBytes(payload)
 	if err != nil {
+		metrics.ErrorTokenUpdating()
```
Comment (absorbb) [inline server/authorization/service.go:187]: Better rename both to ErrorAPIKeyUpdating SuccessAPIKeyUpdating.
That is how we used to call this entity

File: server/metrics/destinations.go (hunk 1 above)
Comment (absorbb) [reply server/metrics/destinations.go:17]: Many things may happens with destinations and sources. it is not clear success of what.
that is why i propose change naming to: init_success, init_error

**Comment by absorbb** [comment]
- ✅ app start / shutdown / ~~running_time~~ - its ok to bw used instead of start / shutdown
- ✅ unit initialization (~~destination~~, ~~source~~, ~~api_key~~, etc) ~~success | error~~
- ✅  ~~new unit created (destination, source, api_key, etc) success | error~~ not needed
- unit test connection (destination, source) success | error
- task_executor.go: source(collection) sync run success | error | cancel | stale?
- uploader.go: file processed (break by destination & table ) success | error
- configurator backend auth status success | login_failed | error
- open API endpoint statuses http codes - enable gin metrics for configurator (copy code from server)
- ✅ mail send status ~~success | error~~
- ✅ slack notification send status ~~success | error~~
- ✅ replay status ~~success | error~~

File: server/metrics/destinations.go (hunk 1 above)
Comment (Alessar) [reply server/metrics/destinations.go]: Renamed

File: server/authorization/service.go (hunk 2 above)
Comment (Alessar) [reply server/authorization/service.go]: Renamed

**Pull request merged**